client/upload-image-to-ipfs-and-return-cid.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+
+from py_client.ipfs import Ipfs
+import click
+import tkinter as tk
+from tkinter import filedialog
+import os
+
+BUSINESSES_PATH = './test-data/bazaar/'
+
+@click.command()
+@click.option('-l', '--ipfs_local', is_flag=True, help='if set, local ipfs node is used.')
+def upload_image(ipfs_local):
+    """
+    Register a business on chain
+
+    :param name: path to LocalBusiness.json with all infos specified in https://github.com/encointer/pallets/blob/master/bazaar/README.md
+    :return:
+    """
+
+    root = tk.Tk()
+    root.withdraw()
+
+    image_title = 'Select your image'
+    bizImageFile = filedialog.askopenfile(mode='r', title=image_title)
+
+    if bizImageFile:
+        logo_path = os.path.abspath(bizImageFile.name)
+        try:
+            image_cid = Ipfs.add(logo_path, ipfs_local)
+        except:
+            print("failed to add image to ipfs")
```

Comment:
please exit with an error code and don't just print an error.

Reply:
see client.py for examples how to do that

Reply:
I've seen how you check the return codes and depending on them raise a specific error. 
in our case we do ipfs.add, which either adds remotely or locally. 
In any case we are in an elevated level, so the error would emerge from deeper in code. 
So for the curl command (remote ipfs add) there are 96 error return codes. (https://everything.curl.dev/usingcurl/returns) 
any of them are actually already handled below, we just need to add the "check" option where the curl command gets excecuted.
I'm not sure I understand if you want me to handle those errors in the ipfs class by myself, because those errors are actually already handled by the subprocess.run function https://github.com/python/cpython/blob/main/Lib/subprocess.py#L528

Reply:
just define a positive integer number which isn't used already and throw that. Don't care what the exact error was deeper down. just ensure we exit with a positice exit code